app/src/views/features/sessions/SessionViewer/SavedSessionViewer.tsx

```diff
@@ -111,11 +111,16 @@ const SavedSessionViewer: React.FC = () => {
     });
   }, [id, eventsFilePath, navigateToList]);
 
+  const hasUserCreatedSessions = useMemo(
+    () => userAttributes?.num_sessions_saved_online > 0 || userAttributes?.num_sessions_saved_offline > 0,
+    [userAttributes?.num_sessions_saved_online, userAttributes?.num_sessions_saved_offline]
+  );
+
```

Comment:
Existing users who have seen the setting banner may see it once again. Can we also add a check for `num_sessions` here.